jst-nuke-boilerplate-push

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# Pulls Boilerplate into each package and attempts to push the changes up.
+# Copyright © 2018 RobLoach
+# MIT-licensed
+
+[ -f config.ini ] && \
+root=$(awk -F '=' '{if (! ($0 ~ /^;/) && $0 ~ /root/) print $2}' config.ini)
+
+cwd="`pwd`"
+
+for d in `echo ${root}jstransformer*/`; do
+  repo="`basename $d`"
+  echo "Processing $repo"
+  cd "$d"
+  git add -A
+  git commit -m "Update Boilerplate"
```

Comment:
Should the commit message still be "Update Boilerplate"? Should there be something to skip the commit if there are no changes?

Reply:
Since `--allow-empty` isn't set, empty commits arn't made.

Reply:
👍